app/models/audit/ChangePlatformOperatorAuditEventModel.scala

```diff
@@ -0,0 +1,197 @@
+    val registeredBusinessAddressInUk = info.addressDetails.countryCode.map{ countryCode =>
+      if (Country.ukCountries.map(_.code).contains(countryCode)) {Json.obj("registeredBusinessAddressInUk" -> true)} else {
+        Json.obj("registeredBusinessAddressInUk" -> false)}
+    }.getOrElse(Json.obj())
+    val addressLine2 = info.addressDetails.line2.map {line2 => Json.obj("addressLine2" -> line2)}.getOrElse(Json.obj())
+    val city = info.addressDetails.line3.map {line3 => Json.obj("city" -> line3)}.getOrElse(Json.obj())
+    val region = info.addressDetails.line4.map {line4 => Json.obj("region" -> line4)}.getOrElse(Json.obj())
+    val postCode = info.addressDetails.postCode.map {postCode => Json.obj("postCode" -> postCode)}.getOrElse(Json.obj())
+    val countryCode = info.addressDetails.countryCode.map {countryCode => Json.obj("countryCode" -> countryCode)}.getOrElse(Json.obj())
+    val country = info.addressDetails.countryCode.flatMap { countryCode => Country.allCountries.find(_.code == countryCode).map(c => c.name)}
+    val countryName = info.addressDetails.countryCode.map {_ => Json.obj("country" -> country)}.getOrElse(Json.obj())
+    val registeredBusinessAddress =
+      Json.obj(
+        "businessAddress" -> Json.obj(
+          "addressLine1" -> info.addressDetails.line1
+        ).++(addressLine2).++(city).++(region).++(postCode).++(countryCode).++(countryName)
+      )
+    registeredBusinessAddressInUk ++ registeredBusinessAddress
+  }
+
+  private def getContactJsonUPO(info: UpdatePlatformOperatorRequest): JsObject = {
+    val canPhonePrimaryContactJson = info.primaryContactDetails.phoneNumber.map(phoneNumber => Json.obj(
+      "canPhonePrimaryContact" -> true,
+      "primaryContactPhoneNumber" -> phoneNumber
+    )).getOrElse(Json.obj("canPhonePrimaryContact" -> false))
+    val primaryContactDetails = Json.obj(
+      "primaryContactName" -> info.primaryContactDetails.contactName,
+      "primaryContactEmail" -> info.primaryContactDetails.emailAddress,
+    ) ++ canPhonePrimaryContactJson
+    val hasSecondaryContactJson = if (info.secondaryContactDetails.nonEmpty) {Json.obj("hasSecondaryContact" -> true)}
+    else {Json.obj("hasSecondaryContact" -> false)}
+    val canPhoneSecondaryContactJson = info.secondaryContactDetails.map(secondaryContact =>
+      secondaryContact.phoneNumber.map( phone =>
+        Json.obj(
+          "canPhoneSecondaryContact" -> true,
+          "secondaryContactPhoneNumber" -> phone
+        )).getOrElse(Json.obj("canPhoneSecondaryContact" -> false))
+    ).getOrElse(Json.obj())
+    val secondaryContactDetails = info.secondaryContactDetails.map(details => Json.obj(
+      "secondaryContactName" -> details.contactName,
+      "secondaryContactEmail" -> details.emailAddress,
+    )).getOrElse(Json.obj()) ++ canPhoneSecondaryContactJson
+    primaryContactDetails ++ hasSecondaryContactJson ++ secondaryContactDetails
+  }
+
+}
```

Comment:
Would have been better to have model Class. But happy for now as we have in same other Repo.